simple_history/templatetags/getattributes.py

```diff
@@ -0,0 +1,12 @@
+from django import template
+
+register = template.Library()
+
+
+def getattribute(value, arg):
+    """Gets an attribute of an object dynamically from a string name"""
+
+    if hasattr(value, str(arg)):
+        return getattr(value, arg)
```

Comment:
why do you call `str` on `arg` in `hasattr`? you don't do it with `getattr`
also you don't need that `if hasattr(...):` in the first place
```py
def get_attribute(value, arg):
    return getattr(value, arg, None)
```